internal/pubsub/pubsub.go

```diff
@@ -206,8 +217,12 @@ func NewSubscription(d driver.Subscription) *Subscription {
 			id := m.ackID
 			ids = append(ids, id)
 		}
+		callCtx := context.TODO()
```

Comment:
Should there be an issue to go with this context.TODO? Do you really mean context.Background?

Reply:
Done.